adapters/sovrn/sovrntest/supplemental/invalid-adunitcode.json

```diff
@@ -61,6 +61,7 @@
               },
               "tagid": "123456",
               "ext": {
+                "AdUnitCode": "sovrn_auc",
```

Comment:
Interesting that this test was missing an invalid ad unit code. However, based on the json schema, this looks like a valid code. What makes it invalid?

Reply:
@sejanick I suggest deleting this test to reduce confusion. `adunitcode.json` already tests that values in `imp.ext` are passed through.

Reply:
Makes sense - done.